src/encoded/tests/data/workbook-inserts/file_format.json

```diff
@@ -15,7 +15,8 @@
         "identifier": "BAM",
         "standard_file_extension": "bam",
 		"extra_file_formats": [
-			"BAI"
+			"BAI",
+            "VCF"
```

Comment:
Consider reformatting

Reply:
Seems like there's some debate as to whether a VCF is an "extra file format" on Slack? Not sure if it's relevant, but thought I'd let you know just in case it is: https://hms-dbmi.slack.com/archives/C051155U7QC/p1706202607799989

Reply:
Ah just saw your message! Ignore the above.